### PR TITLE
[docs] generate parameters description from config file. Stage 1

### PR DESCRIPTION
**Pull request by StrikerRUS**
This is the continuation of #1381.

@guolinke It seems that `helper\parameter_generator.py` cannot handle links in params' descriptions (`desc` and `descl2` fields) due to this:

https://github.com/Microsoft/LightGBM/blob/28832d9e55e016d02c157e541293b39b18dc44af/helper/parameter_generator.py#L27

Can we hotfix it? Maybe three forward slashes?

### CODE REVIEW

**Comment by guolinke** [comment]
@StrikerRUS 
refer to my solution here: https://github.com/Microsoft/LightGBM/blob/master/helper/parameter_generator.py#L27-L29 

you can join [1:]

**Comment by StrikerRUS** [comment]
@guolinke 
Sorry, didn't get it.

Did you mean this:
```
tokens = '//'.join(line.split("//")[1:]).split("=")
```

**Comment by guolinke** [comment]
@StrikerRUS yeah, Or just
```line.strip()[2:].stripe()``` ,
which can remove the first two `//` .

**Comment by StrikerRUS** [comment]
@guolinke OK, thank you!

File: include/LightGBM/config.h [hunk 1]
```diff
@@ -76,126 +78,125 @@ struct Config {
...
 
   // [doc-only]
-  // alias=random_seed
-  // desc=Use this seed to generate seeds for others, e.g. data_random_seed.
-  // desc=Will be override if set other seeds as well
-  // default=none
+  // alias = random_seed
+  // desc = this seed is used to generate other seeds, e.g. ``data_random_seed``, ``feature_fraction_seed``
+  // desc = will be overridden, if you set other seeds
   int seed = 0;
```
Comment (StrikerRUS) [inline include/LightGBM/config.h]: Why default=none?

Comment (guolinke) [reply include/LightGBM/config.h]: I think it is a typo.

Comment (StrikerRUS) [reply include/LightGBM/config.h]: I had the same thoughts, so deleted it.

What is your opinion about it https://github.com/Microsoft/LightGBM/pull/1409#pullrequestreview-125173261 ?

Comment (StrikerRUS) [reply include/LightGBM/config.h]: Rolled back in #1872.

**Pull request merged**